### DISCUSSION TITLE
SilkSpawners Compactibility

### DISCUSSION THREAD

**Discussion by DinikCZ** [Zrips/CMI#538] [Feature Requests]
I'm using SilkSpawners for ShopGui+, since players are able to buy spawners.
I want to use Spawner menu feature in CMI, but there is a problem. When i FullDisable spawners, i can't use that menu, only Spawner command. 
When i enable Spawners in CMI config, placed SilkSpawners are transformed into Pig Spawners.

Preview of that (propably) bug: https://i.imgur.com/VTXyqX6.mp4

Cmi Version (using/cmi version): **7.8.6.0**
Server Type (Spigot/Paperspigot/etc): **PaperSpigot**
Server Version (using /ver): **git-Paper-1468 (MC: 1.12.2)**

**Comment by TheWranger** [comment]
I've asked the developer of ShopGui+ to add support for CMI spawners.
https://www.spigotmc.org/threads/shopgui-admin-gui-shop-plugin-paid.63389/page-108#post-3033112
CMI support in ShopGui+ would be a better solution to this issue imo.
